packages/react/src/components/CodeSnippet/CodeSnippet-story.js

```diff
@@ -48,12 +50,23 @@ const props = {
   }),
 };
 
+const lightPropMessage = (
+  <small style={{ display: 'block', paddingBottom: '1rem' }}>
+    {'Make sure to use a different background color for the '}
```

Comment:
This copy is unclear - we should clarify that they should be using the prop with certain themes

Reply:
Hey @jillianhowarth the prop is currently available regardless of theme. I'm happy to change the copy though -- what should it be?

Reply:
It's safe to use with all themes actually. It's working correctly in the White and Gray 10 themes as well. 

If we want to get more specific/technical we could say something like: 
- The snippet container should never be the same color as the page background. 
- Do not use the 'light` variant on `$ui-background` or `$ui-02`.